frontend/talentsearch/src/js/api/poolAdvertisementOperations.graphql

```diff
@@ -0,0 +1,47 @@
+mutation createPoolAdvertisement(
+  $userId: ID!
+  $poolAdvertisement: CreatePoolAdvertisementInput!
+) {
+  createPoolAdvertisement(userId: $userId, poolAdvertisement: $poolAdvertisement) {
+    id
+  }
+}
+
+mutation updatePoolAdvertisement(
+  $id: ID!
+  $poolAdvertisement: UpdatePoolAdvertisementInput!
+) {
+  updatePoolAdvertisement(id: $id, poolAdvertisement: $poolAdvertisement) {
+    id
+  }
+}
+
+mutation publishPoolAdvertisement(
+  $id: ID!
+  $poolAdvertisement: PublishPoolAdvertisementInput!
+) {
+  publishPoolAdvertisement(id: $id, poolAdvertisement: $poolAdvertisement) {
+    isPublished
+  }
+}
+
+mutation extendPoolAdvertisementClosingDate(
+  $id: ID!
+  $newExpiryDate: DateTimeTz!
+) {
+  extendPoolAdvertisementClosingDate(id: $id, newExpiryDate: $newExpiryDate) {
+    expiryDate
+  }
+}
+
+mutation closePoolAdvertisement($id: ID!) {
+  closePoolAdvertisement(id: $id) {
+    expiryDate
+  }
+}
```

Comment:
I get this error when I try this mutation, and the extend mutation:
```
Add [expiry_date] to fillable property to allow mass assignment on [App\\Models\\Pool].
```

Reply:
Ah yes I need to add that to the fillable attribute in the Pool model.